### DISCUSSION TITLE
Integrate terra 2.0 x/alliance module for Classic

### DISCUSSION THREAD

**Discussion by nghuyenthevinh2000** [classic-terra/core#165] [Ideas]
Let's connect two moons together

**Comment by mysolo** [comment]
Does this remain compatible with a future integration of interchaine security to allow terra classic to validate other ICS compatible chains?

**Reply by nghuyenthevinh2000** [reply]
yes, we can make it compatible

**Reply by mysolo** [reply]
If terra classic can manage both, it opens the way to much more possibilities and interaction with all the different cosmos chains 
I vote for it :)

**Reply by lunctoken** [reply]
It's a good idea, can we relaunch this topic?
